pkg/generator/template.go

```diff
@@ -12,6 +12,7 @@ var tpl = template.Must(template.New("generated").
 package {{.Package}}
 
 import (
+	"fmt"
```

Comment:
Looks like the rest of the imports use 4 spaces rather than tabs. It's not great, but let's stay consistent for now?